### DISCUSSION TITLE
What kinda of sick joke is this?

### DISCUSSION THREAD

**Discussion by glowsoony** [KadeArchive/Kade-Engine#2465] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/84847356/136434804-6801a878-dbde-4ff2-86ab-44fc78ebeaff.png)

I DIDNT DO THIS! i started up 1.7 kade engnie adding my character and this happend. btw my character is not "bf-pixel"
its bf-pixelcourrpted but this image has bf-pixel normal version so idk what to do.

**Comment by Jaduplex** [comment, marked as answer]
i think you forgot to copy over the pixel rescaling code.

**Reply by glowsoony** [reply]
Tysm!

**Comment by isakube** [comment]
smol boifend
